test/integration/singlecluster/controller/core/suite_test.go

```diff
@@ -87,3 +88,36 @@ func managerSetup(ctx context.Context, mgr manager.Manager) {
 	failedCtrl, err := core.SetupControllers(mgr, queues, cCache, controllersCfg)
 	gomega.Expect(err).ToNot(gomega.HaveOccurred(), "controller", failedCtrl)
 }
+
+func managerSetupWithWorkloadRetentionEnabled(ctx context.Context, mgr manager.Manager) {
```

Comment:
Can we propagate Configuration and use managerSetup? Like we did in Jobs.

https://github.com/kubernetes-sigs/kueue/blob/ba42340c6467669a454d0a267140e4fc1a4f3e4d/test/integration/singlecluster/controller/jobs/job/suite_test.go#L89

It would also be helpful to use different configurations for different tests.